de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/browser/BrowserAction.java

```diff
@@ -0,0 +1,30 @@
+package de.fu_berlin.inf.dpp.whiteboard.ui.browser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class BrowserAction {
+
+    private String type;
+    private String elementType;
+    private String id;
+    private ArrayList<Map<String, String>> attributes;
```

Comment:
Is always null, the class do not seem to be finished yet.

Reply:
Can you please describe this behavior in the class description ? And if GSON is capable of instantiating classes with private CTORs, then please make the CTOR also private if this class is never instantiated manually.

Reply:
done